app/errors/check_never_run_error.rb

```diff
@@ -1,7 +1,16 @@
 # frozen_string_literal: true
 
 class CheckNeverRunError < StandardError
-  def initialize(msg = "The requested check was never run against this ref, exiting...")
-    super
+  def initialize(all_checks)
+    names = all_checks.empty? ? ["No checks"] : all_checks.map{|x| x.name}
```

Comment:
We could add some better message when no checks were found running against the ref. 
```suggestion
    return super("No checks were found funning against this ref, exiting...")
    
    names = all_checks.map(&:name)
```